packages/vuetify/src/composables/items.ts

```diff
@@ -93,12 +93,16 @@ export function useItems (props: ItemProps) {
   const items = computed(() => transformItems(props, props.items))
 
   function transformIn (value: any[]): InternalItem[] {
-    return value.map(v => {
-      const existingItem = items.value.find(item => deepEqual(v, item.value))
-      // Nullish existingItem means value is a custom input value from combobox
-      // In this case, use transformItem to create an InternalItem based on value
-      return existingItem ?? transformItem(props, v)
-    })
+    return value
+      // When the model value is null, returns an InternalItem based on null
+      // only if null is one of the items
+      .filter(v => v != null || (v == null && items.value.some(item => item.value == null)))
```

Comment:
You don't need to check if `v == null` because it's implied. Same feedback as other null comparisons.

Reply:
thanks heaps, removed `v == null` and only check if null is one of the items

Reply:
Refactored, it should be as simple as always only checking if model value is one of items